main fix nsis exe

SurfaceS opened the pull request:
# Description of code changes

- removed `heapmem` from setup file and exe file
- added setup file properties `ProductName`, `CompanyName`, `Version`, etc
- added details about creating the uninstaller instead of leaving the last copied file, which makes the user think it takes a while

# Checklist
- [ ] Any relevant issues are [referenced](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#referencing-issues-and-pull-requests)
- [ ] Any relevant [Knowledge Base](https://github.com/UniversalMediaServer/knowledge-base) articles are written/modified
- [ ] Any relevant tests have been written/modified

SubJunk commented:
@SurfaceS after your comment in the Docker thread about Java managing its memory better after JRE 11, I was wondering whether we should default to no heapmem, what do you think?

SurfaceS commented:
- removed `heapmem` from setup file and exe file
- added setup file properties `ProductName`, `CompanyName`, `Version`, etc
- added details about creating the uninstaller instead of leaving the last copied file, which makes the user think it takes a while

SubJunk commented:
Do you think this should merge into v15, in case there are problems? Then we can get feedback on the stability via the v15 testers

SurfaceS commented:
@SubJunk this one was merged into v15 (no mem limit anymore).

but we still need to fix at least the `exe` bug. see #5390
